systemctl

```diff
@@ -16,16 +16,53 @@ systemctl stop foo.service
 # To reload a service's configuration:
 systemctl reload foo.service
 
+# To edit a service's configuration:
+systemctl edit foo.service
+
+# To reload systemd manager configuration:
+systemctl daemon-reload
+
 # To enable a service to startup on boot:
 systemctl enable foo.service
 
 # To disable a service to startup on boot:
 systemctl disable foo.service
 
+#To see all active units, add --all for everything
```

Comment:
This line needs a space between the `#` and `To`, and is missing the `:` at the end as well. Same for the lines below.